ntbs-service/Startup.cs

```diff
@@ -350,14 +350,16 @@ private void ConfigureHangfire(IApplicationBuilder app)
             app.UseHangfireDashboard("/hangfire", dashboardOptions);
             app.UseHangfireServer(new BackgroundJobServerOptions { WorkerCount = 1 });
             GlobalJobFilters.Filters.Add(new AutomaticRetryAttribute { Attempts = 0 });
-            
+
             if (!Env.IsDevelopment())	
             {
```

Comment:
We can remove this conditional (and maybe move the explanation text?) given we are controlling it through entries in `appsettings.Development.json` now?